protos/bond.proto

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message Bond {
+   enum CouponType {
+      FIXED = 0;
+      FLOATING = 1;
+   }
+
+   enum CouponFrequency {
+      QUARTERLY = 0;
+      MONTHLY = 1;
+      DAILY = 2;
+   }
+
+   // Identifier of the bond (ISIN)
+   string bond_id = 1;
+
+   // Ticker symbol of the organization which issued the bond
+   string issuer = 2;
+
+   // A key/value map of rating agency to rating
+   map<string, string> corporate_debt_ratings = 3;
+
+   // The yearly rate of interest paid in coupons
+   float coupon_rate = 4;
+
+   // If floating, the interest rate is calculated by adding coupon-rate
+   // to the rate looked up using the benchmark index (Libor)
+   CouponType coupon_type = 5;
+
+   // How often the coupons are paid out
+   CouponFrequency coupon_frequency = 6;
+
+   // Date the first coupon can be paid for the bond
+   // Format: MM/DD/YYYY
+   string first_coupon_date = 7;
+
+   // Date on which the bond becomes worth it’s face value and can be redeemed
+   // Format: MM/DD/YYYY
+   string maturity_date = 8;
+
+   uint64 amount_outstanding = 9;
+
+   // Par-value of the bond, to be paid upon maturity
+   string face_value = 10;
+
+   // Date on which the bond was first settled
+   // Format: MM/DD/YYYY
+   string first_settlement_date = 11;
+}
+
```

Comment:
Just to confirm, was it on purpose that we went away with the field [`coupon-benchmark`](https://sawtooth.hyperledger.org/docs/core/releases/0.7/bond_family_guide/design/object_model/bond.html#attributes)?  https://sawtooth.hyperledger.org/docs/core/releases/0.7/bond_family_guide/design/object_model/bond.html#attributes

Reply:
Our intention with not including the coupon-benchmark parameter was based on the fact that the attribute, if set, must be set to 'Libor' and is only required if the bond has a floating coupon type. Therefore, to us, it seemed unnecessary.

Reply:
Makes sense! I just wanted to double-check it was intentional and not a forgotten field.